_i18n/en/_docs/footprint.md

```diff
@@ -16,6 +16,14 @@ FRIDA_OBJC_BRIDGE ?= auto
 FRIDA_SWIFT_BRIDGE ?= auto
 {% endhighlight %}
 
+If working on embedded systems, all the previous features may be disabled. 
+Specifically, they are only required in the following cases:
+- FRIDA_V8: default Javascript runtime is QuickJS so it can safelly disabled if not used. Required if the v8 runtime is needed, for example when specifically requested via the API `create_script(..., runtime='v8')` or in the CLI with `--runtime=v8`.
+- FRIDA_CONNECTIVITY: required if using certificates to enable TLS, or if using `setup_peer_connection()` (API) or `--p2p` (CLI). Note that it is not required for network connectivity. For example, it is not required when using frida-server as `frida-server -l 0.0.0.0`. 
+- FRIDA_DATABASE: required if using the [SqliteDatabase](https://frida.re/docs/javascript-api/#sqlitedatabase) and related APIs, can be safely disabled if not.
+- FRIDA_JAVA_BRIDGE: required only if using in a Java Virtual Machine or Android Runtime environment. Note that there are other languages appart from Java which runs either on the JVM or the Android Runtime, such as Kotlin and Scala. 
+- FRIDA_OBJC_BRIDGE and FRIDA_SWIFT_BRIDGE: required if either the istrumented app or the system daemons / APIs are programmed using Objective C or Swift respectively. Usually required in Apple OSes, like IOS and macOS, may be safely disabled outside the Apple ecosystem.
```

Comment:
```suggestion
If working on embedded systems, all the aforementioned features may be disabled.

Specifically, they are only required in the following cases:
- FRIDA_V8: default Javascript runtime is QuickJS so it can safely disabled if
  not used. Required if the V8 runtime is needed, for example when specifically
  requested via the API `create_script(..., runtime='v8')` or through the
  frida-tools CLI with `--runtime=v8`.
- FRIDA_CONNECTIVITY: required if using certificates to enable TLS, or if using
  `setup_peer_connection()` (API) or `--p2p` (CLI). Note that it is not required
  for network connectivity. For example, it is not required when using
  frida-server like this: `frida-server -l 0.0.0.0`.
- FRIDA_DATABASE: required if using [SqliteDatabase]({% link _docs/javascript-api.md#sqlitedatabase %})
  and related APIs, can be safely disabled if not.
- FRIDA_JAVA_BRIDGE: required when wanting to call or instrument Java APIs
  inside processes with a Java Virtual Machine or Android Runtime environment.
  Note that there are other languages apart from Java which run either on the
  JVM or the Android Runtime, such as Kotlin and Scala.
- FRIDA_OBJC_BRIDGE and FRIDA_SWIFT_BRIDGE: required when wanting to call or
  instrument Objective-C or Swift code. Useful on Apple OSes, like i/macOS, may
  be safely disabled outside the Apple ecosystem.
```